.github/workflows/aro-hcp-dev-env-cd.yml

```diff
@@ -1,5 +1,11 @@
 ---
   name: ARO HCP Dev Environment Continuous Deployment
+  env:
+    REGION: westus3
+    REGIONAL_RESOURCEGROUP: aro-hcp-dev-westus3
+    SC_RESOURCEGROUP: aro-hcp-dev-westus3-sc
+    MC_RESOURCEGROUP: aro-hcp-dev-westus3-mc-1
```

Comment:
i suppose `sc` is for service cluster and `mc` is for management cluster?
We should have a naming convention already. @nwnt can provide the doc.
Nonetheless, even we will only have one svc cluster, it's possible we will have multiple of them for various migration reason. so keeping the numbering suffix will make it consistent and future proof.

Reply:
these are just the names for our DEV env. we can adopt the naming pattern of microsoft though. @bennerv @mjlshen